### DISCUSSION TITLE
post/10-all-in-quickconnect/

### DISCUSSION THREAD

**Discussion by giscus** [lyc8503/lyc8503#12] [Announcements]
# post/10-all-in-quickconnect/

群晖的 QuickConnect 是群晖自带的一项内网穿透功能, 通过某宝的洗白服务, 自建的黑群晖也可以使用 QuickConnect 来进行内网穿透. QuickConnect 与群晖自家的应用搭配使用很方便, 可以实现内外网的无缝切换, 只需要填入 QuickConnect ID 即可. 但缺点是 QuickConnect 功能只能在群晖官方的 APP 上使用, 而群晖的很多 APP 实际使

https://blog.lyc8503.site/post/10-all-in-quickconnect/

**Comment by ZHH-start** [comment]
hmmm，小猫咪是啥……

**Reply by lyc8503** [reply]
是使用了 Clash 内核的代理客户端...

**Reply by ZHH-start** [reply]
懂了，我正好在用，感谢。让我试试部署看，官方的打洞速度太慢，只有100k多点，实在是没法正常用。请问博主现在这个方案还在用吗

**Reply by lyc8503** [reply]
我这边还在作为备用方案, 因为最近发现了这个 https://blog.lyc8503.site/post/ipv4-tcp-hole-punching/

之前使用的时候最低有 10Mbps, 最高能到 30-40 Mbps, 感觉还算挺充足的. 可能和运营商、地区也有关.

**Comment by ZHH-start** [comment]
收到，我在底下回了


哈哈公主
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "lyc8503/lyc8503"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年10月28日(星期六) 凌晨1:43
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [lyc8503/lyc8503] post/10-all-in-quickconnect/ (Discussion #12)





 
我这边还在作为备用方案, 因为最近发现了这个 https://blog.lyc8503.site/post/ipv4-tcp-hole-punching/
 
之前使用的时候最低有 10Mbps, 最高能到 30-40 Mbps, 感觉还算挺充足的. 可能和运营商、地区也有关.

**Comment by jackyangming** [comment]
更新系统要不要重新配置呀

**Reply by lyc8503** [reply]
我不太清楚，因为群晖7不能洗白QC，群晖6也没有后续更新了

**Comment by fycyx** [comment]
感谢大佬，已经按照步骤完成了访问内网的服务，中间踩了一些坑比如v2ray两个版本不一致导致的无法连接。有一些问题想问下大佬。1.我现在是win上面建立与nas的隧道同时浏览器使用SwitchyOmega的socket5代理可以正常访问，有没有更加其他更加优雅的方式。2.我本来想要尝试使用clash或者v2rayn的配置文件，这样就就可以导入以后一键直接访问内网资源。但是好像没法实现。3.安卓上面好像无法使用config.json的配置文件，导致现在一直无法使用手机访问服务器资源。

**Comment by KaosuX** [comment]
话说现在初级版本的那俩文章写的方法现在好像用不了了，我试了下。。不知道是操作有问题还是怎么

**Reply by lyc8503** [reply]
群晖 6 目前还是可行的，参考我文中的配置方法。7 我没有使用，不太清楚。
